helmctlr/helm_test.go

```diff
@@ -137,6 +160,28 @@ func TestResourceAddedHappyPath(t *testing.T) {
 	assertCounters(t, ct, func() {
 		testController.ResourceAdded(testResource)
 	})
+
+	// Ensure helm configMap owner references get set
```

Comment:
This seems fine unless we need to handle additional states (see previous comments)